### PR TITLE
[proxima-beam-testing] #143 add TestStreamStorage

### PR DESCRIPTION
**Pull request by je-ik**

### CODE REVIEW

File: beam/core-testing/src/main/java/cz/o2/proxima/beam/storage/TestStreamStorage.java [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  /**
+   * Replace all storages in all attribute families in given config with {@code test-stream}.
+   *
+   * @param config the configuration to replace
+   * @return replaced config
+   */
+  public static Config replaceStorages(Config config) {
+    final Map<String, String> overrides = new HashMap<>();
+    config
+        .getObject("attributeFamilies")
```
Comment (LesTR) [inline beam/core-testing/src/main/java/cz/o2/proxima/beam/storage/TestStreamStorage.java:69]: Can we move this as constant into `ConfigConstants`?

File: beam/core-testing/src/main/java/cz/o2/proxima/beam/storage/TestStreamStorage.java [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+        .unwrapped()
+        .forEach(
+            (key, value) -> {
+              @SuppressWarnings("unchecked")
+              final Map<String, Object> cast = (Map<String, Object>) value;
+              Preconditions.checkState(cast.containsKey("storage"));
```
Comment (LesTR) [inline beam/core-testing/src/main/java/cz/o2/proxima/beam/storage/TestStreamStorage.java:75]: `ConfigConstants.STORAGE`

File: beam/core-testing/src/main/java/cz/o2/proxima/beam/storage/TestStreamStorage.java [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+              overrides.put(
+                  String.format("attributeFamilies.%s.storage", key),
```
Comment (LesTR) [inline beam/core-testing/src/main/java/cz/o2/proxima/beam/storage/TestStreamStorage.java:77]: same

File: beam/core-testing/src/main/java/cz/o2/proxima/beam/storage/TestStreamStorage.java (hunk 1 above)
Comment (je-ik) [reply beam/core-testing/src/main/java/cz/o2/proxima/beam/storage/TestStreamStorage.java:69]: Already did in parallel. :-)

File: beam/core-testing/src/main/java/cz/o2/proxima/beam/storage/TestStreamStorage.java (hunk 2 above)
Comment (je-ik) [reply beam/core-testing/src/main/java/cz/o2/proxima/beam/storage/TestStreamStorage.java:75]: :+1:

File: beam/core-testing/src/main/java/cz/o2/proxima/beam/storage/TestStreamStorage.java (hunk 3 above)
Comment (je-ik) [reply beam/core-testing/src/main/java/cz/o2/proxima/beam/storage/TestStreamStorage.java:77]: :+1:

**Comment by je-ik** [comment]
This closes O2-Czech-Republic#143

**Pull request merged**